plugins/crowdin/src/App.css

```
@@ -0,0 +1,127 @@
+/* Your Plugin CSS */
+
+main {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    padding: 0 15px 0;
+    height: 100%;
+    gap: 15px;
+
+    user-select: none;
+    -webkit-user-select: none;
+}
+.form-field {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 10px;
+    position: relative;
+}
+
+.loader {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: var(--framer-color-bg);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-Index: 10;
+}
+
+.loading {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+}
+
+.logo {
+    color: var(--framer-color-text-tertiary);
+    background-color: var(--framer-color-bg-tertiary);
+    border-radius: 8px;
+}
+
+.setup {
+    align-items: center;
+}
+
+.setup select {
+    font-weight: 600;
+}
+
+.setup img {
+    height: 180px;
+    width: 100%;
+    border-radius: 10px;
+    background-color: #263238;
+    object-fit: contain;
+}
+
+.setup button {
+    margin-top: 5px;
+}
+
+.setup img {
+    user-select: none;
+    -webkit-user-select: none;
+    pointer-events: none;
+}
+
+.setup label {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 30px;
+    width: 100%;
+    justify-content: space-between;
+    color: var(--framer-color-text-secondary);
+    padding-left: 15px;
+}
+
+.setup select {
+    width: 100%;
+}
+
+.setup input,
+.setup select {
+    width: 134px;
+}
+
+.setup {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    gap: 15px;
+    width: 100%;
+}
+
+.setup {
+    padding-bottom: 15px;
+}
+
+a {
+    color: currentColor;
+}
+.button-stack {
+    display: flex;
+    gap: 8px;
+    justify-content: space-between;
+}
+.button-stack button {
+    flex: 1;
+}
+.crowdin-button-primary {
+    color: var(--framer-color-text-reversed);
+    background-color: #263238;
+}
```

Combine duplicate .setup rules and add an empty line between each rule.

```suggestion
.setup {
    width: 100%;
    display: flex;
    flex-direction: column;
    align-items: start;
    gap: 15px;
    width: 100%;
    padding-bottom: 15px;
}

a {
    color: currentColor;
}

.button-stack {
    display: flex;
    gap: 8px;
    justify-content: space-between;
}

.button-stack button {
    flex: 1;
}

.crowdin-button-primary {
    color: var(--framer-color-text-reversed);
    background-color: #263238;
}

```